core/templates/pages/contributor-dashboard-admin-page/contributor-dashboard-tables/contributor-admin-stats-table.component.html

```diff
@@ -386,7 +386,7 @@ <h2>{{ noDataMessage }}</h2>
   </table>
   <div class="dashboard-list-footer" *ngIf="!loadingMessage && !noDataMessage && !checkMobileView()">
     <div class="dashboard-list-footer-count">
-      <span>Displaying {{ statsPageNumber * itemsPerPage + 1 }} - {{ getUpperLimitValueForPagination() }} of many</span>
+      <span>Displaying {{ statsPageNumber * itemsPerPage + 1 }} - {{ getUpperLimitValueForPagination() }} of {{ more ? 'many' : getUpperLimitValueForPagination() }} </span>
```

Comment:
Why do we still want show **many**?

Reply:
@chris7716 We are not getting total number of records from backend, so it will require changes in backend.

Reply:
Could you please find a solution to send total number of records from the backeend?